packages/react-components/react-charts-preview/library/src/components/Legends/shape.tsx

```diff
@@ -41,6 +41,10 @@ export const Shape: React.FunctionComponent<IShapeProps> = React.forwardRef<HTML
         transform={`rotate(${
           shape === Points[Points.diamond] ? 45 : shape === Points[Points.pyramid] ? 180 : 0
         }, 0, 0)`}
+        style={{
```

Comment:
Is this needed when width/height property is already defined.

Reply:
yes, without this the shape is coming huge